### PR TITLE
Writing first test description to double check if test module works.

### PR DESCRIPTION
**Pull request by DigitalSpaceCat**
This kata is good for beginner javascript but also love to see how intermediate groups apply their logic.

### CODE REVIEW

**Comment by DigitalSpaceCat** [comment]
Hi All, I submit new kata. The quiz comes from my book "eloquent javascript". 
This kata is good for beginner javascript but also love to see how intermediate groups apply their logic.

File: katas/chess-board/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "banking-kata",
```
Comment (rodolfoprr) [inline katas/chess-board/package.json]: think this would be `"name": "chess-game"`

**Comment by rodolfoprr** [comment]
cool, thanks @DigitalSpaceCat :D

**Comment by DigitalSpaceCat** [comment]
Hello @rodolfoprr I changed the package name

**Pull request merged**